packages/web-components/src/utils/table.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx React.createElement */
+import React from 'react';
+
+/**
+ * This is based off of the React `<Table>` implementation where fields
+ * was an array of objects w/ `label`, `value`, and a few other properties.
+ * For this we only care about `label`.
+ */
+interface FieldType {
+  label: string;
+}
+
+/**
+ * Useful for generating child content for a `<va-table>`
+ *
+ * @param {array} data - An array of arrays, where the inner arrays represent row data
+ * @param {array} fields - The field/column names
+ */
+export function generateTableChildren(data: any[], fields: FieldType[]): any[] {
+  return [
+    /** @ts-ignore */
```

Comment:
The `@ts-ignore` comments are because without the `h` JSX factory import from `components.d.ts` which also includes `<va-table-row>` as a [JSX "Intrinsic Element"](https://github.com/department-of-veterans-affairs/component-library/blob/ba2bca2c9c1251d4960789addba0ad7659ae93f3/packages/web-components/src/components.d.ts#L1240), we get errors. I tried some different imports and couldn't get it to work, and I didn't want to duplicate code, so I just ignored the warning since it shouldn't be an issue when this function gets used.